File: SkaiLab.Invoice.Models/Vendor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkaiLab.Invoice.Models
{
	public class Vendor
	{
		public long Id { get; set; }
		public string OrganisationId { get; set; }
		public string LegalName { get; set; }
		public string DisplayName { get; set; }
		public int CurrencyId { get; set; }
		public Currency Currency { get; set; }
		public string TaxNumber { get; set; }
		public string BusinessRegistrationNumber { get; set; }
		public string ContactId { get; set; }
		public Contact Contact { get; set; }
		public string LocalLegalName { get; set; }
	}
}

File: SkaiLab.Invoice/ClientApp/src/app/models/vendor.ts
import { Contact } from "./contact";
import { Currency } from "./currency";

export class Vendor{
    id:number;
    organisationId:string;
    legalName:string;
    displayName:string;
    currencyId:number;
    currency:Currency;
    taxNumber:string;
    businessRegistrationNumber:string;
    contactId:string;
    contact:Contact;
    localLegalName:string;
    constructor(){
        this.id=0;
        this.organisationId="";
        this.legalName="";
        this.displayName="";
        this.currencyId=0;
        this.currency=new Currency();
        this.taxNumber="";
        this.businessRegistrationNumber="";
        this.contactId="";
        this.contact=new Contact();
        this.localLegalName="";
    }
}